#this message defines a group of tourist points

tp_msg[] tp_miniverse